modules/ROOT/partials/configuration/format_noneditable_selector.adoc

```diff
@@ -0,0 +1,20 @@
+[[format_noneditable_selector]]
+== `+format_noneditable_selector+`
+
+This option allows a non-editable (`+contenteditable="false"+`) element that matches the selector specified to be wrapped with a valid format.
```

Comment:
This doesn't allow all formats from memory. It only allows inline elements to work IIRC? As you can't use a selector format for example as it's not wrappable.

Reply:
Technically it can apply to non-editable blocks if the format is block type with `wrapper: true`. The two defaults would be applying blockquote and div. Then it can wrap an inline non-editable element with any inline format assuming it is valid as per the schema.

I'll see if I can put something together without being too wordy.

Reply:
Does adding something like this seem suitable?
```
Valid formats are block formats that have the `+wrapper+` parameter set to `+true+` and inline formats. These can be applied to non-editable block elements such as `+<p>+` and non-editable inline elements such as `+<span>+` respectively.

Selector formats are not supported.
```

Reply:
I don't think mentioning anything about `valid formats` is a good idea, as valid formats just means it's a format that TinyMCE accepts. What we really want here is `formats that allow wrapping` or similar.

Reply:
Good point, that would be a better way of phrasing it.

Reply:
I have pushed up some changes. Hopefully, it is closer to want we should say.

Reply:
That seems reasonable to me.